packages/salesforcedx-vscode-core/package.json

```diff
@@ -281,6 +281,10 @@
         }
       ],
       "commandPalette": [
+        {
+          "command": "sfdx.force.package.install",
+          "when": "sfdx:project_opened"
```

Comment:
Does it make sense to also check for the default username state?

Reply:
I don't think we need that, we do those checks at the command level